### ISSUE TITLE
Wrong example in readme

### ISSUE THREAD

**Issue by marcospb19** [udoprog/leaky-bucket#13]
In this example (from the README):

```rust
use leaky_bucket::RateLimiter;
use std::time;

#[tokio::main]
async fn main() {
    let limiter = RateLimiter::builder()
        .max(10)
        .initial(0)
        .refill(5)
        .build();

    let start = time::Instant::now();

    println!("Waiting for permit...");

    // Should take about 5 seconds to acquire in total.
    let a = limiter.acquire(7);
    let b = limiter.acquire(3);
    let c = limiter.acquire(10);

    let ((), (), ()) = tokio::join!(a, b, c);

    println!(
        "I made it in {:?}!",
        time::Instant::now().duration_since(start)
    );
}
```

There is this line that says:

> // Should take about 5 seconds to acquire in total.

But here it took only 300 millis.

Currently the default value is `100` milliseconds, but [it used to be 1 second](https://github.com/udoprog/leaky-bucket/blame/3554f2739ee276100bab9b5e03c35b18b67858a0/src/lib.rs#L281).

Is that example outdated?

When I insert `.interval(std::time::Duration::from_secs(1))` in the builder, it gives me 3 seconds.

**Comment by udoprog** [comment]
Ah, good catch (and sorry for missing this issue!). It should indeed say 400ms.

**Comment by udoprog** [comment]
Closed in #14
